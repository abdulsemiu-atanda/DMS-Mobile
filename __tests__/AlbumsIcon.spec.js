import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import AlbumsIcon from '../components/icons/AlbumsIcon.react'

describe('AlbumsIcon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AlbumsIcon tintColor='blue' />).toJSON()

    expect(tree).toMatchSnapshot()
    expect(tree.props.style[0].color).toEqual('blue')
  })
})
