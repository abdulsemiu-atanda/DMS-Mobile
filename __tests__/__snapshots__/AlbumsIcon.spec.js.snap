// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumsIcon renders correctly 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": "blue",
        "fontSize": 32,
      },
      undefined,
      Object {
        "fontFamily": "Ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
    ]
  }
>
  
</Text>
`;
